### PR TITLE
Updating pom for mvn site information/

### PR DESCRIPTION
**Pull request by bartenbach**
Site now correctly displays developer information

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,15 +9,37 @@
 	</parent>
 	<groupId>cist4830.unomaha</groupId>
 	<artifactId>tempo</artifactId>
-	<version>0.0.2</version>
+	<version>0.0.3</version>
```
Comment (malycole) [inline pom.xml]: Should this be version 0.0.4 since this is milestone 4?

Comment (silvercorked) [reply pom.xml]: hmm, we could honestly call it 1.0.0 i think, since milestone 4 ought to be when we are done

Comment (bartenbach) [reply pom.xml]: We can change it to whatever.  The meat of this PR was just the site/docs.

**Review by silvercorked — APPROVED** [review]
looks good

**Review by malycole — APPROVED** [review]
re-approving

**Pull request merged**